API/Models/Issue.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace API.Models
+{
+    public class Issue
```

Comment:
What is this for?

Reply:
Placeholder, for now, tested some fancy things, will be edited later.